docker/examples/sensorthings/README.md

```diff
@@ -1,31 +1,51 @@
-# Pygeoapi with SensorThings (STA)
+# Setting up a Pygeoapi Server with SensorThings (STA) Using Docker Compose
```

Comment:
Good opportunity to replace `Pygeoapi` with `pygeoapi`! Can we adjust to:

Setting up a pygeoapi server with SensorThings (STA) using Docker Compose

Reply:
+1

Reply:
@shivathapaa do you want to make the proposed change?

Reply:
Thank you for the feedback! I have made changes accordingly. :)